renderer/components/FormattedMarkdownMessage.js

```diff
@@ -35,23 +35,16 @@ const remarkReactComponents = {
   a: OpenLinkInBrowser
 }
 
-class FormattedMarkdownMessage extends React.Component {
-  render() {
-    const { formatMessage } = this.context.intl
-    const { id, defaultMessage, values, description } = this.props
-    const messageDescriptor = { id, defaultMessage, description }
-    const message = formatMessage(messageDescriptor, values)
-    // When the message is the ID means it's not defined and we just emit not
-    // element
-    if (message === id) {
-      return null
-    }
-    return remark().use(reactRenderer, {remarkReactComponents}).processSync(message).contents
+const FormattedMarkdownMessage = ({ id, defaultMessage, values, description }) => {
+  const { formatMessage } = useIntl()
```

Comment:
Using the hook instead of the context. Doesn't make a huge difference, just my new-found fascination for hooks :-)

Reply:
what is the difference?

Reply:
It means that instead of defining a class on which you access the `this.context` part of the instance in the render() method you define a pure function on which you call useIntl() which returns a hook.

This hook allows you to make changes to the state of the component without needing to define it as a class.

Effectively the result of this it makes the amount of boilerplate required to do some set of common tasks easier. This feature exists since react 16.8, see: https://reactjs.org/docs/hooks-intro.html.